translations/messages-en.properties

```diff
@@ -968,4 +968,6 @@ setup.statistics.description = Allow anonymous statistics to be submitted to Med
 No\ submission = Do not share impact data
 Submit\ via\ web = Internet only
 Submit\ via\ sms = SMS only
-Submit\ via\ either = Try internet first, then SMS
\ No newline at end of file
+Submit\ via\ either = Try internet first, then SMS
+facility.deleted = [deleted]
+facility.unavailable = [unavailable]
```

Comment:
Don't use the word "facility" - it's a deprecated term for "place". Either use `place.deleted` or make it more generic and use `doc.deleted`.